elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/AsyncTest.java

```diff
@@ -33,7 +35,25 @@
         statements = "INSERT INTO ArtifactGroup (name, commonName, description, deprecated) VALUES\n"
                 + "\t\t('com.example.repository','Example Repository','The code for this project', false);")
 @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD,
-        statements = "DELETE FROM ArtifactVersion; DELETE FROM ArtifactProduct; DELETE FROM ArtifactGroup;")
+        statements = "DELETE FROM ArtifactVersion; DELETE FROM ArtifactProduct; DELETE FROM ArtifactGroup;"
+                + "DELETE FROM Stats; DELETE FROM PlayerStats;")
+@Sql(statements = {
+        "DROP TABLE Stats IF EXISTS;",
+        "CREATE TABLE Stats(id int, measure int, dimension VARCHAR(255));",
+        "INSERT INTO Stats (id, measure, dimension) VALUES\n"
+                + "\t\t(1,100,'Foo'),"
+                + "\t\t(2,150,'Bar');"
+})
+@Sql(statements = {
+        "DROP TABLE PlayerStats IF EXISTS;",
```

Comment:
YOu shouldn't need to add all these DROP/CREATE TABLE statements

Reply:
I have moved these CREATE and DELETE tables into a SQL file and then each test can call it in the `@SQL` Annotation as a script.